packages/nimble-components/build/generate-workers/rollup.config.js

```diff
@@ -0,0 +1,13 @@
+import nodeResolve from '@rollup/plugin-node-resolve';
```

Comment:
@rajsite and @atmgrifter00 and I talked offline. We think the right fix for this confusion would be to move the web worker code and build to a separate package that nimble-components depends on. This would also help solve some other concerns like:
1. guaranteeing Nimble and clients are all using the same version of comlink
2. avoiding possible circular dependencies between the component and the web worker code
3. clearer dependencies between build and source which prevents issues like linting/testing/storybook not working because code is in unexpected locations

However, setting up a new package is a non-trivial amount of work and requires a little bit of Nimble repo expertise. @rajsite volunteered to do this but it would be done in a follow up PR. We'd let this PR go in to unblock follow up work but expect it to be refactored shortly after. In the meantime you should still address the feedback above and elsewhere in the PR to clean things up as best we can before adding a new package.

Seem like a good plan?

Reply:
I wrote a short README.md inside generate-workers, tried to make it similar to the other generate-* directories READMEs. Regarding the second comment, yeah, I understand and it's good to hear that our workers infrastructure will receive a more professional approach😁